tests/Main.hs

```diff
@@ -227,23 +238,53 @@ testGetServiceHealth = testCase "testGetServiceHealth" $ do
 testHealth :: TestTree
 testHealth = testGroup "Health Check Tests" [testGetServiceHealth]
 
+-- Name of existing health check we can rely on for tests that use the Session API
+serfHealth :: Text
+serfHealth = "serfHealth"
+
+-- list of names of service/node checks (which should exist in consul already)
+checkIds :: [Text]
+checkIds = [serfHealth]
+
+-- for requests to session API
+lockDelay :: Maybe a
+lockDelay = Nothing
+
 {- Session -}
 testCreateSession :: TestTree
 testCreateSession = testCase "testCreateSession" $ do
   client@ConsulClient{..} <- newClient
-  let req = SessionRequest Nothing (Just "testCreateSession") Nothing ["serfHealth"] (Just Release) (Just "30s")
+  let ttl = "30s"
+      req = SessionRequest lockDelay
+                           (Just "testCreateSession")
+                           localNode
+                           checkIds
+                           (Just Release)
+                           (Just ttl)
   result <- createSession client req Nothing
   case result of
     Just _ -> return ()
     Nothing -> assertFailure "testCreateSession: No session was created"
 
+
+oneSecond :: Int
+oneSecond = 1 * 1000
```

Comment:
For plain numbers that have some unit of time, length, etc. I usually put that info right into the name to minimise the chance of confusing, e.g.

```haskell
oneSecondMillis :: Int
oneSecondMillis = 1 * 1000
```

(Same for `twelveSeconds` below.)

I'd do it similarly for values like `durationMillis`, `distanceMeters` and so on.

The need for such unit suffixes goes away of course if you have some strong type for the unit, like `Time` or `Distance` that makes it unambiguous.

Reply:
Yes, that's great, I'll update those!

Reply:
Updated.